### DISCUSSION TITLE
All atoms are displayed twice in redux devtools

### DISCUSSION THREAD

**Discussion by PrettyCoffee** [pmndrs/jotai#1795] [Q&A]
Hi!

Soo, I am not sure if I am doing something wrong or if that's a bug so I wanted to have it confirmed first before creating a bug ticket.

When using `useAtomDevtools` on multiple atoms in `react 18` with `vite`, all atoms are displayed twice in the `redux devtools` dropdown:

![image](https://user-images.githubusercontent.com/66521452/221385521-be4eaf93-3005-42f2-8163-94b6ae018733.png)

Not sure why this is, I think I followed the instructions of the docs.
I created a demo in codesandbox to show off the bug:
https://codesandbox.io/p/sandbox/angry-brahmagupta-kpf2zz?file=%2Fsrc%2FDebugAtoms.tsx

In the demo even worse: Changes are not being detected. I don't have this problem in my local project, so no idea what happened there.

Can someone enlighten me? :)

**Comment by dai-shi** [comment]
Does it only happen with StrictMode?
There might be a bug or something. Help wanted.

**Reply by PrettyCoffee** [reply]
Oh yeah, you are right. Without `<StrictMode>` it appears to not happen anymore.

**Reply by PrettyCoffee** [reply]
Seems like excluding it from the `<StrictMode>` fixes the duplicates problem for now:

```tsx
createRoot(root).render(
  <>
    <DebugAtoms />
    <StrictMode>
      <App />
    </StrictMode>
  </>
);
```

**Reply by PrettyCoffee** [reply]
I think I got a fix for the problem, is there a workflow on how I can test it? :)

**Reply by dai-shi** [reply]
Is it https://github.com/jotaijs/jotai-devtools, right? I'm not sure how comprehensive test code is, but there are some tests.

**Reply by PrettyCoffee** [reply]
I created a PR for this:
https://github.com/jotaijs/jotai-devtools/pull/32
